src/metabase/automagic_dashboards/core.clj

```diff
@@ -972,6 +643,10 @@
        (remove (comp nil? second))
        (into {})))
 
+(defn- find-rule-for-entity
```

Comment:
what is a "rule" in this context? I don't think this is a good name given I have no idea what it refers to =)

Reply:
It's the YAML. We often use the term "heuristic" but that feels like it's leaking implementation into the vocabulary. Open to suggestions though

Reply:
what's a heuristic?

Reply:
to be less glib, if you're loading a bunch of models, you should call those models.

if you're loading x-ray definitions, you should call them x-ray definitions.

Both "rules" and "heuristics" are too ambiguous

Reply:
/resources/automagic_dashboards/table/Transactions.yaml 🤷‍♂️

Reply:
Fair point. The terminology is a leftover from the first implementation, but yes, names need love/refactoring too